packages/core/src/components/html-table/_overrides.scss

```diff
@@ -7,7 +7,7 @@
 $section-border-width: 2 * $table-border-width;
 
 table.#{$ns}-html-table {
-
+  $table-border-color: $g-light-base5;
```

Comment:
That's ok, but in order to keep the strucuture of other customizations, we should move our definition of `$table-border-color` & `$dark-table-border-color` to a `_customs.scss` file under `html-table` folder.
Then we should import that `_customs.scss` file, at the begining of both `_html-table.scss` & _overrides.scss`